xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3590,15 +3590,17 @@ void CVideoPlayer::SetSpeed(float speed)
   if (!CanSeek())
     return;
   
-  m_newPlaySpeed = speed * DVD_PLAYSPEED_NORMAL;
+  int iSpeed = static_cast<int>(speed * DVD_PLAYSPEED_NORMAL); //! @todo
```

Comment:
Oooh, I remember - I wasn't sure if I should cast or round. As it is, a speed of 0.98 or -0.98 will be converted to 0. We should probably just use a float throughout, which requires more extensive changes. Feedback?